README.md

```diff
@@ -135,6 +135,23 @@ And you can supply screenshots by creating directories with the following names,
 
 Note that these will replace the current images and screenshots on the play store listing, not add to them.
 
+## Changelogs (What's new)
+
+You can add changelog files under the `changelogs/` directory for each language. The filename should exactly match the version code of the apk that it represents. `supply init` will populate these files from existing Google Play if no metadata exists. 
```

Comment:
Ah, I see. Something like...
"
`supply init` will populate changelog files from existing data on Google Play if no `metadata/` directory exists when it is run.
"

As a separate thought, it would be nice to be able to get your changelog data for the first time even if you already have your other metadata from Google Play...